java/ingest/ingest-status-store/src/main/java/sleeper/ingest/status/store/job/DynamoDBIngestJobStatusStoreCreator.java

```diff
@@ -48,4 +52,11 @@ public static void create(InstanceProperties properties, AmazonDynamoDB dynamoDB
                         new KeySchemaElement(UPDATE_TIME, KeyType.RANGE)));
         configureTimeToLive(dynamoDB, tableName, EXPIRY_DATE);
     }
+
+    public static void tearDown(InstanceProperties properties, AmazonDynamoDB dynamoDBClient) {
+        if (!properties.getBoolean(INGEST_STATUS_STORE_ENABLED)) {
+            return;
+        }
+        dynamoDBClient.deleteTable(DynamoDBIngestJobStatusStore.jobStatusTableName(properties.get(ID)));
```

Comment:
Can we add a log line before this call to deleteTable to log that we're deleting it?